### PR TITLE
Support logging in from Azure IAM

### PR DESCRIPTION
**Pull request by agilob**
Azure responds to Aviatrix client using OPTIONS, because why not

### CODE REVIEW

File: avpnc [hunk 1]
```diff
@@ -45,13 +45,18 @@ class Handler(BaseHTTPRequestHandler):
         return
 
     def do_POST(self):
+        self.extract_response_and_login()
+
+    def do_OPTIONS(self):
+        self.extract_response_and_login()
```
Comment (christophgysin) [inline avpnc]: You could just:
```suggestion
        return self.do_POST()
```

File: avpnc [hunk 2]
```diff
@@ -45,13 +45,18 @@ class Handler(BaseHTTPRequestHandler):
...
+
+    def extract_response_and_login(self):
         self.send_response(200)
         self.send_header('Access-Control-Allow-Origin', '*')
         self.end_headers()
         self.wfile.write(b'SuccessAviatrix')
-        global credentials
-        credentials = json.loads(unquote(self.path[1:]))
-
+        global credentials = json.loads(unquote(self.path[1:]))
```
Comment (christophgysin) [inline avpnc:58]: Why do you need to `unquote()`? Can you provide a (sanitized) example response?

File: avpnc [hunk 3]
```diff
@@ -45,13 +45,18 @@ class Handler(BaseHTTPRequestHandler):
...
+        openvpn(credentials['Email'], credentials['Token'])
```
Comment (christophgysin) [inline avpnc]: This should not be needed.

**Review by christophgysin — CHANGES_REQUESTED** [review]

**Review by christophgysin — APPROVED** [review]

**Comment by christophgysin** [comment]
Thanks!

**Comment by agilob** [comment]
Alright, I see you're the AUR maintainer too, would you mind updating AUR please?

**Comment by christophgysin** [comment]
Done, thanks for the reminder.

**Pull request merged**